documentation/operations-manual.md

```diff
@@ -1 +1,36 @@
-# Travelcy Operations Manual
\ No newline at end of file
+# Travelcy Operations Manual
+
+## Overview
+Travelcy is a native android application developed in Kotlin with Android Studio.
+
+## Coding conventions
+We have been following Kotlins' standard coding convention, which can be found here https://kotlinlang.org/docs/reference/coding-conventions.html
+
+## Testing
```

Comment:
Would be good to have information on how to run the tests in Android Studio and how the tests are run in CI.